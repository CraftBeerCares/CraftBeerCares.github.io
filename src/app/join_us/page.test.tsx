import { render } from "@testing-library/react";
import Page from "./page";

it("renders Join Us page", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
