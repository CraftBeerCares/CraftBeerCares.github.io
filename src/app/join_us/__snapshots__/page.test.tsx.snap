// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Join Us page 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <button
        class="burgerMenuButton"
      >
        Menu
      </button>
      <h1
        class="className title"
      >
        Craft Beer Cares
      </h1>
      <nav
        class="topNav className"
      >
        <a
          class="navLink"
          href="/"
        >
          Home
        </a>
        <a
          class="navLink"
          href="/about"
        >
          About
        </a>
        <a
          class="navLink"
          href="/join_us"
        >
          Join Us
        </a>
        <a
          class="navLink"
          href="/events"
        >
          Events
        </a>
        <a
          class="navLink"
          href="/shop"
        >
          Shop
        </a>
        <a
          class="navLink"
          href="/contact"
        >
          Contact
        </a>
      </nav>
    </header>
    <main
      class="mainContent"
    >
      Join Us
    </main>
    <footer
      class="footer"
    >
      <div
        class="footerNav"
      >
        <nav
          class="footerLinks"
        >
          <nav
            class="className"
          >
            <a
              class="navLink"
              href="/"
            >
              Home
            </a>
            <a
              class="navLink"
              href="/about"
            >
              About
            </a>
            <a
              class="navLink"
              href="/join_us"
            >
              Join Us
            </a>
            <a
              class="navLink"
              href="/events"
            >
              Events
            </a>
            <a
              class="navLink"
              href="/shop"
            >
              Shop
            </a>
            <a
              class="navLink"
              href="/contact"
            >
              Contact
            </a>
          </nav>
        </nav>
        <div
          class="socialLinks"
        >
          <a
            href="https://www.facebook.com/craftbeercares"
          >
            <img
              alt="Facebook"
              data-nimg="1"
              decoding="async"
              height="30"
              loading="lazy"
              src="/assets/facebook.svg"
              style="color: transparent;"
              width="30"
            />
          </a>
          <a
            href="https://www.instagram.com/craftbeercares/"
          >
            <img
              alt="Instagram"
              data-nimg="1"
              decoding="async"
              height="30"
              loading="lazy"
              src="/assets/instagram.svg"
              style="color: transparent;"
              width="30"
            />
          </a>
          <a
            href="https://twitter.com/craftbeercares"
          >
            <img
              alt="Twitter"
              data-nimg="1"
              decoding="async"
              height="30"
              loading="lazy"
              src="/assets/twitter.svg"
              style="color: transparent;"
              width="30"
            />
          </a>
        </div>
        <div
          class="footerQuote"
        >
          Craft Beer Cares - Drinking beer for good Stuff since 2017
        </div>
      </div>
      <div
        class="footerLogo"
      >
        <img
          alt="Craft Beer Cares Logo"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          src="/assets/logo_bw.svg"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; filter: invert(1);"
        />
      </div>
      <div
        class="footerSignUp"
      />
    </footer>
  </div>
</div>
`;
